shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -1278,6 +1278,60 @@ mod protect {
     }
 }
 
+fn test_batch_statements(connection: &Session) {
+    // setup keyspace and table for the batch statement tests
+    {
+        run_query(connection, "CREATE KEYSPACE batch_keyspace WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };");
+        run_query(connection, "CREATE TABLE batch_keyspace.batch_table (id int PRIMARY KEY, lastname text, firstname text);");
+    }
+
+    {
+        let mut batch = Batch::new(BatchType::LOGGED);
+        for i in 0..2 {
+            let statement = format!("INSERT INTO batch_keyspace.batch_table (id, lastname, firstname) VALUES ({}, 'text', 'text')", i);
+            batch.add_statement(&stmt!(statement.as_str())).unwrap();
+        }
+        connection.execute_batch(&batch).wait().unwrap();
+        assert_query_result(
+            connection,
+            "SELECT lastname FROM batch_keyspace.batch_table WHERE id = 1;",
+            &[&[ResultValue::Varchar("text".into())]],
```

Comment:
Can we assert on every row and every field in this table?
I could imagine bugs where some of these get lost in shotovers parsing + reconstruction of batch messages.